### ISSUE TITLE
Handle null disbursement_total values on F3s

### ISSUE THREAD

**Issue by bycoffe** [dwillis/itemizer#12]
The spending field for ActBlue, for example, appears as a path to its schedule B, rather than 0 or blank. disbursement_total is null in the API response. Itemizer should likely have a blank. (Receipts and spending should likely be handled the same way; right now, receipts shows 0.)

![Image of ActBlue filing row](http://f.cl.ly/items/2b0M161b1b2J2w3d0u2x/Screen%20Shot%202015-04-20%20at%202.58.04%20PM.png)

**Comment by dwillis** [comment]
Yeah, this will get fixed once we move NYT app to new infrastructure - right now we're not even loading the filing because of its size. Probably should special-case them for Itemizer, because loading the filing using Fech on Heroku will crash the app.
